cli/test/e2e/package_authoring_e2e_test.go

```diff
@@ -876,6 +998,12 @@ spec:
 		const pkgDir = "./carvel-artifacts/packages/testpackage.corp.dev/"
 		promptOutput := newPromptOutput(t)
 
+		switch testcase.Name {
+		case "Local dir Flow":
+			err := createContentForLocalDir(workingDir)
```

Comment:
I think we can do this outside the loop.

Reply:
To do it outside the loop, I will not be able to cleaup `workingDir` once the test case is finished. And I think it make sense to cleanup everything once testcase is finished. What do you think?

Reply:
> I will not be able to cleaup workingDir once the test case is finished

I see. Maybe we can always create it then? (Would that cause a conflict in other test cases)
I feel that the increase in number of conditionals for each test case is an indication that we shouldn’t use table tests in this case 🤔

Reply:
I think its better to move it out from table test case. Dont want to create something which wont be used by other TC.